src/natcap/invest/carbon.py

```diff
@@ -94,16 +94,16 @@
                 "A table that maps the land-cover IDs to carbon pools.  "
                 "The table must contain columns of 'LULC', 'C_above', "

```

Comment:
I believe the first required col name is inaccurate
```suggestion
                "The table must contain columns of 'lucode', 'C_above', "

```

Reply:
Great catch!